index.js

```diff
@@ -550,29 +497,29 @@ WuiDom.prototype.getClassNames = function () {
  * @returns {boolean}
  */
 WuiDom.prototype.hasClassName = function (className) {
-	return this.getClassNames().indexOf(className) !== -1;
+	return this.rootElement.classList.contains(className);
 };
 
 /**
  * Allows for adding multiples in separate arguments, space separated or a mix
- * @param {...string} className
+ * @param {...string|..string[]} classNames
  */
-WuiDom.prototype.setClassNames = function (className) {
-	if (arguments.length > 1) {
-		className = joinArgumentsAsClassNames('', arguments);
-	}
-
-	this.rootElement.className = uniqueClassNames(className);
+WuiDom.prototype.setClassNames = function (classNames) {
```

Comment:
I would drop the argument name, and embed line 509 into line 511. This variable doesn't really have to exist.